plutus-contract/src/Language/Plutus/Contract/Test.hs

```diff
@@ -390,7 +391,7 @@ waitingForSlot w sl = PredF $ \(_, r) ->
 emulatorLog
     :: forall s e a.
        ()
-    => ([LogMessage EmulatorEvent] -> Bool)
+    => ([LogMessage (EmulatorTimeEvent EmulatorEvent)] -> Bool)
```

Comment:
I don't love the extra ceremony every time we log something. Can we put this all inside `MultiAgent` so the exported `EmulatorEvent` has the time in it already?